src/views/Register.jsx

```diff
@@ -14,27 +18,92 @@ const RegisterContainer = styled(FlexDiv)`
   padding: 4rem 1rem;
 `;
 
-const Register = () => {
+const Register = (props) => {
+  const [email, setEmail] = useState('');
+  const [password, setPassword] = useState('');
+  const [passwordCheck, setPasswordCheck] = useState('');
+  const [termsOfUseCheck, setTermsOfUseCheck] = useState(false);
+
+  const handleEmail = (e) => {
+    setEmail(e.target.value);
+  };
+
+  const handleNewPassword = (e) => {
+    setPassword(e.target.value);
+  };
+
+  const handleConfirmPassword = (e) => {
+    setPasswordCheck(e.target.value);
+  };
+
+  const handleTermsOfUseCheck = () => {
+    setTermsOfUseCheck(!termsOfUseCheck);
+  };
+
+  const handleSubmit = async (e) => {
+    e.preventDefault();
+    if (!email || !password || !passwordCheck || !termsOfUseCheck) {
+      toast.warn(`Tous les champs doivent être renseignés...`, {
+        position: 'top-center',
+        autoClose: 5000,
+        hideProgressBar: false,
+        closeOnClick: true,
```

Comment:
perhaps you don't need the true value because it's true by default : just closeOnClick (same for paseOnHover and draggable).